### PR TITLE
Introduce php cs fixer

### LINKED ISSUE #3: php-cs-fixer
@DeepDiver1975, @PVince81, @karakayasemi   
Any plans to implement php-cs-fixer (in an early stage) as we have in core and the other repros ?

### PR DESCRIPTION
**Pull request by karakayasemi**
closes #3

### CODE REVIEW

**Comment by mmattel** [comment]
I removed myself as reviewer as the others I added have much more experience and knowledge about that than myself.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+# kdevelop
+.kdev
+*.kdev4
+*.kate-swp
+
+# Lokalize
+*lokalize*
+
+# eclipse
```
Comment (phil-davis) [inline .gitignore]: There are lots of duplicate lines added here.
Please review and minimize.

**Comment by phil-davis** [comment]
@karakayasemi I pushed a commit to tell composer to find dependencies that work with PHP 5.6.
The failing Travis jobs were using PHP 5.6 and complaining that composer could not find a valid combination of stuff.

**Comment by karakayasemi** [comment]
@phil-davis thank for your help, I removed duplicated lines from .gitignore

**Comment by phil-davis** [comment]
And I just added the a basic ``make test-php-style`` to the ``Makefile``

There is no "gold standard" ``Makefile`` etc format in use at the moment. So might as well "just do it" here, and the pieces can easily be tidied up as I migrate CI to drone and someone sorts out the "gold standard" ``Makefile``

**Comment by phil-davis** [comment]
Now getting green Travis runs that include passing PHP code style checks.
I will squash commits a bit!

**Comment by phil-davis** [comment]
@patrickjahns @DeepDiver1975 happy to do a first implementation like this? And move to drone etc later. If so, then merge it.

**Comment by patrickjahns** [comment]
I would have loved to get it directly properly setup in `composer-bin`

**Pull request merged**